elastic/integrations
# [cisco_meraki] Migrate log stream visualization to saved search

## Proposed commit message

- Migrated the log stream visualization to saved search
- Re-exported dashboard at 8.13

## Checklist


## How to test this PR locally

```
cd packages/cisco_meraki
elastic-package test system -d log --defer-cleanup 5m

Load up dashboard, ensure that time range covers logs, and view the log stream visualization at the bottom of the dashboard.
```

## Related issues

- Closes #10516

## Screenshots

![Screenshot 2024-12-05 at 10 44 46 AM](https://github.com/user-attachments/assets/a7e0b4de-7f58-4c0d-ae2b-4cc3547a2a0f)
